webBrow/index.js

```diff
@@ -0,0 +1,326 @@
+          button.classList
+            .remove('calculator-button-active')}, 100)
+        break
+      case '5':
+        button.classList
+          .add('calculator-button-active')
+        setTimeout(function(){
+          button.classList
+            .remove('calculator-button-active')}, 100)
+        break
+      case '6':
+        button.classList
+          .add('calculator-button-active')
+        setTimeout(function(){
+          button.classList
+            .remove('calculator-button-active')}, 100)
+        break
+      case '7':
+        button.classList
+          .add('calculator-button-active')
+        setTimeout(function(){
+          button.classList
+            .remove('calculator-button-active')}, 100)
+        break
+      case '8':
+        button.classList
+          .add('calculator-button-active')
+        setTimeout(function(){
+          button.classList
+            .remove('calculator-button-active')}, 100)
+        break
+      case '9':
+        button.classList
+          .add('calculator-button-active')
+        setTimeout(function(){
+          button.classList
+            .remove('calculator-button-active')}, 100)
+        break
+      case '+':
+        button.classList
+          .add('calculator-button-right-active')
+        setTimeout(function(){
+          button.classList
+            .remove('calculator-button-right-active')}, 100)
+        break
+      case '-':
+        button.classList
+          .add('calculator-button-right-active')
+        setTimeout(function(){
+          button.classList
+            .remove('calculator-button-right-active')}, 100)
+        break
+      case '*':
+        button.classList
+          .add('calculator-button-right-active')
+        setTimeout(function(){
+          button.classList
+            .remove('calculator-button-right-active')}, 100)
+        break
+      case '/':
+        button.classList
+          .add('calculator-button-right-active')
+        setTimeout(function(){
+          button.classList
+            .remove('calculator-button-right-active')}, 100)
+        break
+      case 'Clear':
+        button.classList
+          .add('calculator-button-clear-active')
+        setTimeout(function(){
+          button.classList
+            .remove('calculator-button-clear-active')}, 100)
+        break
+      case '.':
+        button.classList
+          .add('calculator-button-bottom-right-active')
+        setTimeout(function(){
+          button.classList
+            .remove('calculator-button-bottom-right-active')}, 100)
+        break
+      case '=':
+        button.classList
+          .add('calculator-button-bottom-right-right-active')
+        setTimeout(function(){
+          button.classList
+            .remove('calculator-button-bottom-right-right-active')}, 100)
+        break
+    }
+  }
+
+  document.querySelectorAll('.calculator-body').forEach(function(element){
+    var kal = new Calculator(element)
```

Comment:
Go ahead and merge this pull request to see if it fixes the calculator operators. Good work!